homeassistant/components/climate/spider.py

```diff
@@ -0,0 +1,143 @@
+"""
+Support for Spider thermostats.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.spider/
+"""
+
+import logging
+
+from homeassistant.components.climate import (
+    ATTR_TEMPERATURE, STATE_COOL, STATE_HEAT, STATE_IDLE,
+    SUPPORT_TARGET_TEMPERATURE, SUPPORT_OPERATION_MODE, ClimateDevice)
+from homeassistant.components.spider import DOMAIN
```

Comment:
This platform belongs to the climate domain. So we're not allowed to import another domain as `DOMAIN`. Import the spider domain like so:
```py
from homeassistant.components.spider import DOMAIN as SPIDER_DOMAIN
```